templates/user/auth/link_account.tmpl

```diff
@@ -4,15 +4,15 @@
 		<div class="new-menu-inner">
 			<!-- TODO handle .ShowRegistrationButton once other login bugs are fixed -->
 			{{if not .AllowOnlyInternalRegistration}}
-				<a class="item {{if not .user_exists}}active{{end}}"
+				<div class="item {{if not .user_exists}}active{{end}}"
```

Comment:
This change is not right.

`<a>` has the ability to accept a11y "Enter" to "click". See "aria/dropdown.js"

Reply:
Actually, this change is completely unrelated and landed in here accidentally.
I originally planned to convert all `href`-less `<a>` at once, but decided on splitting it into several PRs instead.
Reverting.